webui/react/src/e2e/models/ant/Popover.ts

```diff
@@ -56,6 +56,7 @@ export class Popover extends BaseComponent {
    */
   async open(): Promise<this> {
     await this.openMethod();
+    await this.root._page.waitForLoadState('networkidle');
```

Comment:
this wouldn't apply to every popover, would it? a popover that needs to wait on this should get it's own openMethod or it should perform a wait after opening.

Reply:
I thought I removed networkidle checks so this might not be needed.